src/main/java/co/inventrosoft/hibernate_task/console/ConsoleParser.java

```diff
@@ -39,17 +32,17 @@ public void close() throws IOException {
                     teams.add(team);
                     break;
                 }
-                System.out.println("Team with name \"" + team.getName() + "\" already exists");
+                LOGGER.error("Team with name \"" + team.getName() + "\" already exists");
```

Comment:
If you use Logback (this is default logger in Spring Boot) you should use this approach to format logs:
`log.error("Team with name {} already exists", team.getName())`